### PR TITLE
View step closes #42

### LINKED ISSUE #42: View Step

### PR DESCRIPTION
**Pull request by danielsomekh**

### CODE REVIEW

**Comment by png262** [comment]
looks good

File: browser/js/common/factories/StepsFactory.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ app.factory('StepsFactory', function($http){
 	StepsFactory.getStep = function(stepId){
 		return $http.get('/api/steps/' + stepId)
 		.then(function(response){
+			console.log("response data is ", response.data)
```
Comment (tiffanywang3) [inline browser/js/common/factories/StepsFactory.js]: delete console.logs

File: browser/js/view-step/view-step.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	$stateProvider.state('viewStep', {
+		url: '/view-tutorial/:tutorialId/view-step/:stepId',
+		templateUrl: 'js/view-step/view-step.html',
+		controller: 'ViewStepCtrl',
+		params: {
+			tutorial: null,
+		},
+		resolve: {
+			theStep: function($stateParams, StepsFactory){
+				console.log("in get step")
```
Comment (tiffanywang3) [inline browser/js/view-step/view-step.js]: delete console.log

**Comment by tiffanywang3** [comment]
Take out console.logs (or not, we can do a console.log check at the end and take them all out then!)

**Pull request merged**